pinot-common/src/main/java/org/apache/pinot/sql/parsers/CalciteSqlParser.java

```diff
@@ -49,6 +49,17 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(CalciteSqlParser.class);
 
+  private static final Map<String, String> FUNC_NAME_MAP;
+
+  // Calcite Sqlkind name to pinot operator name map
+  static {
+    FUNC_NAME_MAP = new HashMap<String, String>();
```

Comment:
Could you rename this to `CALCITE_TO_PINOT_FUNC_NAME_MAP`. Also it's better to make this an immutable map.